### PR TITLE
Leaderboard hit  the API

### PR DESCRIPTION
**Pull request by ridaarif98**
- Create a new game by using the `API`.
- Implement the form `Submit` button for sending data to the `API`.
- Implement the `Refresh` button to receive data from the `API` and parsing the `JSON`.
- Use `async` and `await` JavaScript features to consume the API.
- Integrate linter.
- Add description in  `README` file.

### CODE REVIEW

File: src/displayScore.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const getScore = async (url) => {
+  const score = await fetch(url);
+  return score.json();
+};
+
```
Comment (carlos-ssh) [inline src/displayScore.js:5]: Kindly add a script to run your project in your ` package.json `.
- adding this line to your scripts: ` "start": "webpack serve --open", `
- Check your ` package.json ` settings or your ` webpack` configuration.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @ridaarif98 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1 +1,24 @@
...
\ No newline at end of file
+import './style.css';
+import sendScores from './sendScore.js';
+import {
+  getScore, displayScores,
+} from './displayScore.js';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/BT0dBrz8PnCOOMBT0tZ4/scores/';
+const form = document.getElementById('gameForm');
+const refreshBtn = document.getElementById('refreshBtn');
```
Comment (carlos-ssh) [inline src/index.js:9]: - Your refreshBtn it's missing functionallity when you hit the ` GET API ` endpoint causing the scores list to change.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @ridaarif98 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -1 +1,24 @@
...
+const userScore = document.getElementById('scoreText');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  sendScores(url, userName, userScore);
+});
+
+refreshBtn.addEventListener('click', () => {
+  getScore(url).then((data) => { displayScores(data); });
+});
```
Comment (ridaarif98) [inline src/index.js:20]: Here is the functionality of `Refresh` button

Comment (ridaarif98) [inline src/index.js]: thanks for the review. But I add the functionality of the refresh button and it is working. Here is the refrence

File: src/index.js (hunk 2 above)
Comment (ridaarif98) [reply src/index.js:9]: Thanks for the review. But I add the functionality of the refresh button in `index.js`.

File: src/index.js (hunk 3 above)
Comment (henatan99) [reply src/index.js]: - The refresh button is working fine. Thank you for making the changes requested by the previous reviewer. Great work! :dancers:

**Review by henatan99 — APPROVED** [review]
Hi @ridaarif98 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**